src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not directly related to the PR but this was annoying me becuase I couldn't get an overview of the test output